dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/fat/src/com/ibm/ws/jaxws/fat/EnableSchemaValidationTest.java

```diff
@@ -0,0 +1,553 @@
+        String response = runTest(server, "testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=WrongNamespace");
+        // Log response to output.txt
+        LOG.info("testWrongNamespaceGlobalSchemaValidationTrue - Response = " + response);
+
+        assertNull("Expected null response from server, but was" + response, response);
+        assertNotNull("Expected Unmarshalling Error: cvc-complex-type.2.4.a  in server logs",
+                      server.waitForStringInLog("cvc-complex-type.2.4.a"));
+    }
+
+    /**
+     * Test the global setting of enableSchemaValidation=true in server.xml, with the wrong content in the <response> element
+     * The response will contain an addtional child element <WrongElementContent> which will cause an Unexected Element exception
+     * with the default schema validation enabled.
+     *
+     * Server config - global-true-server.xml
+     * Expected response - Unmarshalling Error: unexpected element
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testWrongElementContentGlobalSchemaValidationTrue() throws Exception {
+        server.reconfigureServer("EnableSchemaValidationTestServer/global-true-server.xml", "CWWKG0017I");
+
+        // Visit the URL:  http://hostName:hostPort/testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=user
+        String response = runTest(server, "testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=WrongElementContent");
+        // Log response to output.txt
+        LOG.info("testWrongElementContentGlobalSchemaValidationTrue - Response = " + response);
+
+        assertNull("Expected null response from server, but was" + response, response);
+        assertNotNull("Expected Unmarshalling Error: Expected elements are (none) in server logs",
+                      server.waitForStringInLog("local:\"WrongElementContent\""));
+    }
+
+    // GLOBAL FALSE TESTS
+
+    /**
+     * Test the global setting of enableSchemaValidation=false in server.xml, with an additional element <arg0> following the <response> element
+     * which is ignored when schema validation is disabled.
+     *
+     * Server config - global-false-server.xml
+     * Expected response - Hello, AddedElement
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testAddedElementGlobalSchemaValidationFalse() throws Exception {
+        server.reconfigureServer("EnableSchemaValidationTestServer/global-false-server.xml", "CWWKG0017I");
+
+        // Visit the URL:  http://hostName:hostPort/testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=user
+        String response = runTest(server, "testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=AddedElement");
+        // Log response to output.txt
+        LOG.info("testAddedElementGlobalSchemaValidationFalse - Response = " + response);
+
```

Comment:
Can we test AssertNotNull also here for response, like in previous tests?

Reply:
Since its a positive tests, the servlet returns a response, and its non-null.